### PR TITLE
Move OEIS to literature

### PR DESCRIPTION
**Pull request by EgbertRijke**
This PR moves the OEIS module to the literature

### CODE REVIEW

File: src/literature/oeis.lagda.md [hunk 1]
```diff
@@ -188,3 +175,7 @@ A006369 = map-collatz-bijection
 A046859 : ℕ → ℕ
 A046859 n = ackermann n n
```
Comment (fredrik-bakke) [inline src/literature/oeis.lagda.md:176]: Do we perhaps want to list this as
```suggestion
A046859 n = ackermann n n
  where open import elementary-number-theory.ackermann-function
```

to make the connection to where the function is coming from? On the other hand, `ackermann` is still clickable...

Comment (EgbertRijke) [reply src/literature/oeis.lagda.md:176]: The better way is just to introduce the diagonal of the Ackermann function as its own function in the file about the Ackermann function, and then introduce it here by importing it like the other entries.

Comment (fredrik-bakke) [reply src/literature/oeis.lagda.md:176]: I suppose so. And then just leave the obvious functions as is?

Comment (fredrik-bakke) [reply src/literature/oeis.lagda.md:176]: are you leaving this for a future PR?

Comment (EgbertRijke) [reply src/literature/oeis.lagda.md]: I just did it. I was in the middle of something in my globular pull request so I couldn't immediately switch branches.

**Review by fredrik-bakke — APPROVED** [review]
Looks good to me. Feel free to merge when ready

**Pull request merged**